src/main/java/com/mparticle/model/DeviceInformation.java

```diff
@@ -190,6 +190,51 @@ public static PlatformEnum fromValue(String value) {
   @SerializedName(SERIALIZED_NAME_BLUETOOTH_VERSION)
   private String bluetoothVersion;
 
+  public static final String SERIALIZED_NAME_ATT_TIMESTAMP_UNIXTIME_MS = "att_timestamp_unixtime_ms";
+  @SerializedName(SERIALIZED_NAME_ATT_TIMESTAMP_UNIXTIME_MS)
+  private Long attTimestampUnixtimeMs;
+
+  /**
+   * Gets or Sets attAuthorizationStatus
+   */
+  public enum ATTStatusEnum {
```

Comment:
```suggestion
  public enum ATTStatus {
```

"enum" seems unnecessary. the type tells you that.